DuckDuckGo/DBP/DataBrokerProtectionManager.swift

```diff
@@ -70,4 +70,8 @@ extension DataBrokerProtectionManager: DataBrokerProtectionDataManagerDelegate {
     public func dataBrokerProtectionDataManagerDidDeleteData() {
         loginItemInterface.dataDeleted()
     }
+
+    public func dataBrokerProtectionDataManagerWillOpenSendFeedbackForm() {
+        NotificationCenter.default.post(name: .OpenUnifiedFeedbackForm, object: nil, userInfo: ["source": "pir"])
```

Comment:
💡  We should avoid fixing strings we send and retrieve in another place. What do you think of doing the following?

```swift
enum UnifiedFeedbackSource: String, StringRepresentable {
    case settings, ppro, vpn, pir, itr, unknown
    static var `default` = UnifiedFeedbackSource.unknown

    static func userInfo(source: UnifiedFeedbackSource) -> [String: Any] {
        let sourceKey = "source"
        return [sourceKey: source.rawValue]
    }
}
```

then here we can just do:

```swift
public func dataBrokerProtectionDataManagerWillOpenSendFeedbackForm() {
    NotificationCenter.default.post(name: .OpenUnifiedFeedbackForm, object: nil, userInfo: UnifiedFeedbackSource.userInfo(source: .pir))
}
```

We should follow this change on `IdentityTheftRestorationPagesUserScript.swift` too.

Reply:
Good suggestion, thanks for the code sample 👍